Dockerfile

```diff
@@ -0,0 +1,2 @@
+FROM squidfunk/mkdocs-material
```

Comment:
I would advise to change the file name or the file location.
Currently it looks as if the package in this repo is defined by this docker file.
If we can avoid adding this file and run a more complicated command as part of the docs build process it will be even better, but I'm not familiar with a way to do it.

Reply:
@HarelM This is the way that Material for MkDocs recommends. https://squidfunk.github.io/mkdocs-material/getting-started/?h=docker#with-docker

<img width="831" alt="image" src="https://github.com/maplibre/maplibre-style-spec/assets/649392/a288f9a4-b739-43c1-ac42-cf744ae2399e">